test/Tester/Facet/StorageFeatureAttribute.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using Orleans;
+using Orleans.Runtime;
+
+namespace Tester
+{
+    [AttributeUsage(AttributeTargets.Property)]
```

Comment:
If these attributes weren't restricted to properties, then they could one day be used on ctor params,